docs/development/ci-deployment-workflow.md

```diff
@@ -1,7 +1,5 @@
-
 # CI Deployment Workflow
```

Comment:
This file is not very clear. It's about the deployment of what?

Reply:
Of the components

Reply:
thanks